spec/migrations/20160119125950_add_created_on_for_container_entities_spec.rb

```diff
@@ -0,0 +1,82 @@
+require_migration
+
+def stub_name_for(table)
+  "#{table}_stub"
+end
+
+def stub_for(table)
+  send(stub_name_for(table))
+end
```

Comment:
After you remove the tests for schema changes (columns dropped/added, see below), do we need these methods anymore?  Maybe just do the work in-line if it's only a few places.